dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/programrule/engine/ProgramRuleEngineServiceTest.java

```diff
@@ -86,12 +87,12 @@
 
     private ProgramRuleAction programRuleActionA;
 
-    private List<RuleAction> actions;
+    private List<RuleEffect> ruleEffects;
 
```

Comment:
We could perhaps add a test that makes sure only one notification is scheduled when the same scheduling is attempted twice for the same programinstance.